### DISCUSSION TITLE
Build-in builder pattern with Slonik

### DISCUSSION THREAD

**Discussion by issamwahbi** [gajus/slonik#546] [Ideas]
Hello, 

To start thank you a lot @gajus for this great library. 
I was wondering if the library can support out of the box builder pattern to construct SQL queries. For some simple queries sometimes we don't need to have access to the raw sql. 
I think that having a builder pattern bake in will make the adoption of this library more easy and incremental. Any thourght on that ?

**Comment by gajus** [comment]
Thank you.

It is not something I would build into Slonik (because the optimal API would be highly subjective), but something that can easily be built around Slonik as a community package.
